addons/arcadian/data/kolimator.rvmat

```diff
@@ -0,0 +1,11 @@
+ambient[] = {100.000000,0.500000,0.500000,1.000000};
```

Comment:
What is this file name? We can name it properly.

Also add a comment of the original source (for future comparisons if necessary).

Reply:
Renamed it to `collimator_sight.rvmat` that's essentially all it is.